contracts/AINFTFactory.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/interfaces/IERC721.sol";
+
+import "./AINFT721.sol";
+
+contract AINFTFactory is Ownable {
+    address private _clonedAINFTContract = address(0);
+    address private _createdAINFTContract = address(0);
+
+    constructor() Ownable() {}
+
+    function cloneERC721(
+        ERC721 originalNFT_,
+        string memory newName_,
+        string memory newSymbol_
+    ) public returns (address) {
+        address originalNFTAddr = address(originalNFT_);
+        require(originalNFTAddr != address(0), ""); 
+        
+        address ret = _cloneInstanceAINFT721(originalNFT_, newName_, newSymbol_); // msg.sender is admin
+        require(ret != address(0), "Cloned contract is successfully created");
```

Comment:
The cloned contract is successfully created?

Reply:
Same as above